app/views/rails_performance/javascripts/app.js

```diff
@@ -167,6 +161,179 @@ function showRTChart(div, data) {
+      xAxis: {
+          crosshair: true,
+          type: 'datetime',
+          labels: {
+            style: {
+              color: "#a6b0cf"
+            }
+          }
+      },
+      yAxis: {
+          min: 0,
+          title: {
+              text: '%',
+              style: {
+                color: "#f6f6f6"
+              }
+          },
+          labels: {
+            style: {
+              color: "#a6b0cf"
+            }
+          }
+      },
+      legend: {
+          enabled: false
+      },
+      exporting: {
+        buttons: {
+          contextButton: {
+            theme: {
+              fill: "#eee"
+            }
+          }
+        }
+      },
+      plotOptions: {
+          area: {
+            color: '#ff5b5b',
+          }
+      },
+      series: [{
+          type: 'line',
+          name: name,
+          data: data,
+          fillOpacity: 0.3,
+          lineWidth: 1,
+          states: {
+            hover: {
+              lineWidth: 2
+            }
+          }
+      }]
+  });
+};
+
+
+function showUsageChart(div, data, addon, name) {
+  Highcharts.chart(div, {
+      chart: {
+          type: 'line',
+          zoomType: 'x',
+      },
+      title: {
+          text: ''
+      },
+      tooltip: {
+          borderWidth: 0,
+          backgroundColor: 'yellow',
+          pointFormat: '{point.y}',
+          //headerFormat: '',
+          shadow: false,
+          style: {
+              fontSize: '16px',
+              color: '#000',
+          },
+          formatter: function() {
+              if (this.y == 0) {
+                return false;
+              }
+            return bytes(this.y, true);
+          }
+      },
+      xAxis: {
+          crosshair: true,
+          type: 'datetime',
+          labels: {
+            style: {
+              color: "#a6b0cf"
+            }
+          }
+      },
+      yAxis: {
+          min: 0,
+          title: {
+              text: '',
+              style: {
+                color: "#f6f6f6"
+              }
+          },
+          labels: {
+            style: {
+              color: "#a6b0cf"
+            }
+          }
+      },
+      legend: {
+          enabled: false
+      },
+      exporting: {
+        buttons: {
+          contextButton: {
+            theme: {
+              fill: "#eee"
+            }
+          }
+        }
+      },
+      plotOptions: {
+          area: {
+            color: '#ff5b5b',
+          }
+      },
+      series: [{
+          type: 'line',
+          name: name,
+          data: data,
+          fillOpacity: 0.3,
+          lineWidth: 1,
+          states: {
+            hover: {
+              lineWidth: 2
+            }
+          }
+      }]
+  });
+};
+
+function bytes(bytes, label) {
+  if (bytes == 0) return '';
+  var s = ['bytes', 'KB', 'MB', 'GB', 'TB', 'PB'];
+  var e = Math.floor(Math.log(bytes)/Math.log(1024));
+  var value = ((bytes/Math.pow(1024, Math.floor(e))).toFixed(2));

```

Comment:
`Math.floor` over `e` already present above line

Reply:
oh, it's again some copy-paste from google :)